terraform/templates/manifests/ingress-keycloak.yaml

```diff
@@ -21,14 +21,14 @@ spec:
               service:
                 name: keycloak
                 port:
-                  number: 8081
+                  number: 8080
           - path: /
-            pathType: Exact
+            pathType: Prefix
             backend:
               service:
                 name: keycloak
                 port:
-                  number: 8081
+                  number: 8080
```

Comment:
These are set to 8081 and exact is to make sure the Keycloak admin UI is not accessible from anywhere. E.g. you have to use port-foward to access it. I could have used a SG to block access but it's difficult when I cannot be reasonably sure where users are access accessing ingress from.

Reply:
That makes sense, thank you for the clarification. The issue I was having was with the UI not rendering correctly (missing images), but also not forwarding to the admin ui even with port forwarding.